.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,6 +19,18 @@ If this pull request addresses a Help Wanted ticket, please link the relevant Gi
 Otherwise, link the JIRA ticket.
 -->
 
+#### Screenshots
+<!--
+If the PR includes UI changes, include screenshots/GIFs.
+
+For an easier comparison of UI changes a table (template below) can be used.
+
+|  before  |  after  |
+|----|----|
+| <insert before screenshot here> | <insert after screenshot here> |
```

Comment:
nit 😆 
```md
| Before                          | After                          |
|---------------------------------|--------------------------------|
| <insert before screenshot here> | <after before screenshot here> |
```

Reply:
:D TBH, I have noticed most people just remove that comment and put:

Before:

{SCREENSHOT}

After:

{SCREENSHOT}